app/controllers/mixins/breadcrumbs_mixin.rb

```diff
@@ -74,7 +74,7 @@ def special_page_breadcrumb(variable)
       elsif floating_ip_address?(variable.first)
         title = variable.first[:address]
       else
-        title = variable.first[:name]
+        title = variable.first[:name] if variable.count == 1 # show breadcrumb only for one item
```

Comment:
The condition can be moved to line 70:

```return unless variable && variable.count == 1``` or
```return if !variable || variable.count != 1```

Reply:
Fixed. Chose the second option to get rid of `unless` 😃